app/i18n/fr/conf.php

```diff
@@ -41,6 +41,7 @@
 			'bottom_line' => 'Ligne du bas',
 			'display_authors' => 'Auteurs',
 			'entry' => 'Icônes d’article',
+			'myLabels' => 'My labels',	// TODO
```

Comment:
```suggestion
			'myLabels' => 'Mes étiquettes',
```

We already have the translations for that string, though

Reply:
yes, it is still in draft ;)